.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+# Description
+
+_(Instructions are in italics—please replace them with your own text)_
+
+_Please include a clear and concise summary of the changes. Please also include relevant motivation and context. If you're not familiar with the [XY problem](https://xyproblem.info/), we highly suggest to read the linked article first_ 😄
+
+_Feel free to include additional details and explanations if you think it will help the reviewers, but please respect their time and keep it as brief as you can._
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that breaks backward compatibility)
+- [ ] This change requires a documentation update
+
```

Comment:
Do we want a distinct type of "clean up/non-functional" option e.g. my migration of SDL threads to C++ for Voodoo?

Reply:
Yes, good idea. Adding it.

Reply:
Done.